mlir-sycl/lib/Transforms/Inliner.cpp

```diff
@@ -283,16 +565,37 @@ void Inliner::inlineCallsInSCC(Inliner &Inliner, CallGraphSCC &SCC) {
 
     LLVM_DEBUG(llvm::dbgs() << "** Inline succeeded\n");
```

Comment:
Would it make sense to have a [pass statistic](https://mlir.llvm.org/docs/PassManagement/#pass-statistics) for the number of successfully inlined calls?

Reply:
Added a statistic to count the number of calls inlined. It is turned on via the `-mlir-pass-statistics` flag (like other MLIR statistics).